test/dsl_test.go

```diff
@@ -364,6 +398,19 @@ func notExists(filename string) fileOp {
 	}, Defaults}
 }
 
+func mkfileexcl(name string) fileOp {
+	return fileOp{func(c *ctx) error {
+		dir, file := filepath.Split(name)
+		d, _, err := c.getNode(dir, noCreate, resolveAllSyms)
```

Comment:
It seemed to me that `getNode` was not supposed to error if file existed. But I guess we could treat the `createFileExcl` differently. Will do!